src/MongoDB/Manager.c

```diff
@@ -206,19 +211,22 @@ static void php_phongo_manager_prep_uri_options(zval *options TSRMLS_DC) /* {{{
 				php_phongo_manager_prep_authmechanismproperties(option TSRMLS_CC);
 				continue;
 			}
-		} ZEND_HASH_FOREACH_END();
+		}
+		ZEND_HASH_FOREACH_END();
 	}
 #else
 	{
-		HashPosition   pos;
-		zval         **option;
+		HashPosition pos;
+		zval**       option;
 
-		for (zend_hash_internal_pointer_reset_ex(ht_data, &pos);
-		     zend_hash_get_current_data_ex(ht_data, (void **) &option, &pos) == SUCCESS;
-		     zend_hash_move_forward_ex(ht_data, &pos)) {
-			char  *string_key = NULL;
-			uint   string_key_len = 0;
-			ulong  num_key = 0;
+		for (
```

Comment:
Did `clang-format` add the line break here (akin to `MONGO_DEBUG(` in `php_phongo.c`), or did you?